cli/create.go

```diff
@@ -208,6 +210,13 @@ func (r *RootCmd) create() *clibase.Cmd {
 			Description: "Specify a duration after which the workspace should shut down (e.g. 8h).",
 			Value:       clibase.DurationOf(&stopAfter),
 		},
+		clibase.Option{
+			Flag:        "automatic-updates",
+			Env:         "CODER_WORKSPACE_AUTOMATIC_UPDATES",
+			Description: "Specify automatic updates setting for the workspace (accepts 'always' or 'never').",
+			Default:     string(codersdk.AutomaticUpdatesNever),
+			Value:       clibase.StringOf(&autoUpdates),
+		},
```

Comment:
suggestion: `--enable-automatic-updates` and `clibase.Bool` defaulting to `false`

Reply:
I've deliberately avoided a bool here as we might consider more choices in the future, such as autostart building with the last built version if the new version has incompatible parameters.

Also, at some point when all the dashboard stuff is built that lets you easily turn it off, we'll probably default to "always" which makes an --enable flag awkward to reverse.